aea/cli/utils/package_utils.py

```diff
@@ -133,13 +139,17 @@ def _is_valid_author_handle(author: str) -> bool:
     >>> _is_valid_author_handle("this-is-not")
     ...
     False
+    >>> _is_valid_author_handle("Not_this_one")
+    ...
+    False
+    >>> _is_valid_author_handle("and_not_This_one")
+    ...
+    False
 
     :param author: author name
     :return: bool indicating whether author name is valid
     """
-    if re.fullmatch(PublicId.AUTHOR_REGEX, author) is None:
```

Comment:
dont like check internals exposed and used here.
this code should not know about regex and how to use them for checking